config/200-clusterrole-webhook.yaml

```diff
@@ -15,7 +15,7 @@
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
-  name: triggermesh-webhook
+  name: triggermesh-webhook-namespaced
```

Comment:
Does this role need to be a **Cluster**Role?

Reply:
Well no, it does not have to be a `ClusterRole`. I'll change it